src/main/java/com/selenium/course/pages/ContactForm.java

```diff
@@ -0,0 +1,355 @@
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import com.selenium.course.framework.DriverManager;
+
+/**
+ * Created by reyna ulaque on 13/06/2015.
+ */
+public class ContactForm {
+    private WebDriver driver;
+    private WebDriverWait wait;
+    private Select selectTitleBox;
+    private Select selectLeadSourceBox;
+    private Select selectLevelBox;
+    private SearchWindow searchWindow;
+
+
+    @FindBy(xpath = "//h2[contains(.,' New Contact')]")
+    @CacheLookup
+    WebElement newContactTitle;
+
+    @FindBy(id = "name_salutationcon2")
+    @CacheLookup
+    WebElement titleSelect;
+
+    @FindBy(id = "name_firstcon2")
+    @CacheLookup
+    WebElement firstName;
+
+    @FindBy(id = "name_lastcon2")
+    @CacheLookup
+    WebElement lastName;
+
+    @FindBy(id = "con4")
+    @CacheLookup
+    WebElement accountName;
+
+    @FindBy(xpath = "//img[@alt='Account Name Lookup (New Window)']")
+    @CacheLookup
+    WebElement findAccountLookup;
+
+    @FindBy(id = "con5")
+    @CacheLookup
+    WebElement title;
+
+    @FindBy(id = "con6")
+    @CacheLookup
+    WebElement department;
+
+    @FindBy(id = "con7")
+    @CacheLookup
+    WebElement birthDate;
+
+    @FindBy(id = "con8")
+    @CacheLookup
+    WebElement reportTo;
+
+    @FindBy(id = "con9")
+    @CacheLookup
+    WebElement leadSourceSelect;
+
+    @FindBy(id = "con10")
+    @CacheLookup
+    WebElement phone;
+
+    @FindBy(id = "con13")
+    @CacheLookup
+    WebElement homePhone;
+
+    @FindBy(id = "con12")
+    @CacheLookup
+    WebElement mobile;
+
+    @FindBy(id = "con14")
+    @CacheLookup
+    WebElement otherPhone;
+
+    @FindBy(id = "con11")
+    @CacheLookup
+    WebElement fax;
+
+    @FindBy(id = "con15")
+    @CacheLookup
+    WebElement email;
+
+    @FindBy(id = "con16")
+    @CacheLookup
+    WebElement assistant;
+
+    @FindBy(id = "con17")
+    @CacheLookup
+    WebElement assistantPhone;
+    //Address information
+    @FindBy(id = "con19street")
+    @CacheLookup
+    WebElement mailingStreet;
+
+    @FindBy(id = "con19city")
+    @CacheLookup
+    WebElement mailingCity;
+
+    @FindBy(id = "con19state")
+    @CacheLookup
+    WebElement mailingState;
+
+    @FindBy(id = "con19zip")
+    @CacheLookup
+    WebElement mailingZipPostal;
+
+    @FindBy(id = "con19country")
+    @CacheLookup
+    WebElement mailingCountry;
+
+    @FindBy(id = "con18street")
+    @CacheLookup
+    WebElement otherStreet;
+
+    @FindBy(id = "con18city")
+    @CacheLookup
+    WebElement otherCity;
+
+    @FindBy(id = "con18state")
+    @CacheLookup
+    WebElement otherState;
+
+    @FindBy(id = "con18zip")
+    @CacheLookup
+    WebElement otherZipPostal;
+
+    @FindBy(id = "con18country")
+    @CacheLookup
+    WebElement otherCountry;
+
+    @FindBy(id = "00N1a000005YAYl")
```

Comment:
This locator is dynamic 
Change it by another locator for example css selector or xpath locators